static/app/views/settings/project/projectProcessingIssues.tsx

```diff
@@ -446,6 +447,14 @@ class ProjectProcessingIssues extends Component<Props, State> {
       <div>
         <SentryDocumentTitle title={title} projectSlug={projectId} />
         <SettingsPageHeader title={title} />
+        <Alert type="warning">
+          <TextBlock noMargin>
+            {t(
+              `Processing Issues, along with Legacy Reprocessing, has been deprecated,
+            and will be removed in the future.`
```

Comment:
This is good, I would just inform them of what they should do instead (and they may as well already start) or what to expect in the future. Something like "You will instead be able to manually trigger re-processing from individual issues" or whatever the experience looks like.